src/tests/ftest/util/telemetry_utils.py

```diff
@@ -369,15 +369,15 @@ class TelemetryUtils():
     ENGINE_RANK_METRICS = [
         "engine_rank"]
     ENGINE_NVME_HEALTH_METRICS = [
+        "engine_nvme_commands_read_errs",
+        "engine_nvme_commands_write_errs",
+        "engine_nvme_commands_unmap_errs",
+        "engine_nvme_commands_checksum_mismatch",
```

Comment:
Curious - why move these from below to here?

Reply:
Since I move the position of these entries , so I thought the test will also need to change the order?

Reply:
I don't think it matters, but you could keep them in sync if you want